pkg/tiflash/client.go

```diff
@@ -79,6 +94,40 @@ func (c *Client) SendGetRequest(host string, statusPort int, relativeURI string)
 }
 
 func (c *Client) SendPostRequest(host string, statusPort int, relativeURI string, body io.Reader) ([]byte, error) {
+	err := c.checkValidAddress(fmt.Sprintf("%s:%d", host, statusPort))
+	if err != nil {
+		return nil, err
+	}
+
 	uri := fmt.Sprintf("%s://%s:%d%s", c.httpScheme, host, statusPort, relativeURI)
 	return c.httpClient.WithTimeout(c.timeout).SendRequest(c.lifecycleCtx, uri, http.MethodPost, body, ErrFlashClientRequestFailed, distro.Data("tiflash"))
 }
+
+func (c *Client) getMemebers() ([]topology.StoreInfo, error) {
+	cached, _ := c.cache.Get("tiflash_members")
+	if cached != nil {
+		return cached.([]topology.StoreInfo), nil
+	}
+
+	_, mem, err := topology.FetchStoreTopology(c.pdClient)
```

Comment:
Only TiFlash stores should be allowed.

Reply:
What does this mean? And I updated the variable name. PTAL~